resources/warnings.json

```diff
@@ -6377,7 +6377,7 @@
     "versions": [
       {
         "lastVersion": "1.8.5",
-        "pattern": ".*"
+        "pattern": "1\\.[0-7].*|1\\.8.[0-5]"
```

Comment:
Better never release version 1.10.

I recommend you model your pattern after the existing ones. My recommendation:

    (1[.]7|1[.]8[.][1345])(|[.-].+)|(1[.]8)

Reply:
Okay, I updated the pattern according to your suggestion.